### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#488] [Esami & Esoneri M-Z]
### Traccia

<img width="879" alt="image" src="https://github.com/user-attachments/assets/ec21bdd6-268c-403b-9a83-f3e3c4fba852">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/0c444f06-33b8-4bef-a3ff-d8dc88769d2a)
